packages/editor/src/components/resizable-editor/style.scss

```diff
@@ -16,7 +16,7 @@
 	cursor: ew-resize;
 	outline: none;
 	background: none;
-	border-radius: $radius-block-ui;
+	border-radius: $radius-full;
```

Comment:
I'm assuming we're intentionally going with `radius-full` here?

Reply:
Yes, this is consistent with similar elements in the components package that we want to be fully rounded. E.g: https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/progress-bar/styles.ts#L43